src/lib/chains/algorand/utils.ts

```diff
@@ -0,0 +1,15 @@
+import algosdk from "algosdk";
+
+export const  convertToAscii= (str:string) => {
```

Comment:
it'd be great if you add tests for these methods

Reply:
good suggetion will add